Dockerfile.standalone

```diff
@@ -0,0 +1,30 @@
+FROM ubuntu:16.04
+
+#---install useful tools and dependencies---#
+RUN apt-get update
+RUN apt-get install -y --no-install-recommends \
```

Comment:
It's idiomatic to combine multiple, related shell commands into a single `RUN` statement, separated by `&&` (to ensure that each is run only if the previous succeeds). The main reason for this is to reduce the number of layers in the final image.

I'd probably also include the cleanup command in the `RUN` statement, rather than leaving it till the end of the Dockerfile, to keep it close to the files it's cleaning up.